src/lib/plugins/moderation.js

```diff
@@ -16,7 +16,7 @@ module.exports.server = function (serv, settings) {
     if (!serv.bannedIPs[IP]) {
       serv.bannedIPs[IP] = {
         time: +moment(),
-        reason: reason || 'Your IP is banned!'
+        reason: reason || 'You has been IP banned from this server. Bye!'
```

Comment:
A user seeing this message might not necessarily be the intended target of the IP ban, but is framed as such. The original message is more factually and grammatically correct.

Reply:
I changed it to the vanilla minecraft kick message